src/__testUtils__/genFuzzStrings.ts

```diff
@@ -1,12 +1,10 @@
-// @flow strict
-
 /**
  * Generator that produces all possible combinations of allowed characters.
  */
-export default function* genFuzzStrings(options: {|
-  allowedChars: Array<string>,
-  maxLength: number,
-|}): Generator<string, void, void> {
+export default function* genFuzzStrings(options: {
+  allowedChars: Array<string>;
+  maxLength: number;
+}): Generator<string, void, void> {
```

Comment:
It's so said that TS doesn't support exact types:https://github.com/microsoft/TypeScript/issues/12936
We totally lose all the effort that we spend on making our types exact 😭

Reply:
You are right, but there are some ways to workaround that, see: https://www.typescriptlang.org/play/?#code/C4TwDgpgBAogHgQwMbADwBUA0UBKA+KAXinSgjmAgDsATAZ1wCgooB+XMi6+k5lt3vygAuKFQgA3CACc+o8VNmMAZgFcqKAJYB7KlAQY8ACgBGrUfGRosUAN4BfPAEpRCKiDuNG9rwiMOnAG4oAHoQqG0Aa0Y-WygTUQBGKHsnUPCZaW1ZPypVABt8oPSyaSycozSwiMigA